indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -566,6 +567,23 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_avg_per_question_per_track(self):
+        ratings = {}
+        abstract_ratings = (AbstractReviewRating.query
```

Comment:
`.join(AbstractReviewRating.question)` and then add `~AbstractReviewQuestion.is_deleted` to the filters